website/package.json

```diff
@@ -55,5 +54,9 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "npx": "^10.2.2",
+    "yarn-audit-fix": "^10.0.5"
```

Comment:
Regarding the audit, we currently use Dependabot for checking vulnerabilities in dependencies. However, I personally don't have the permissions to view those. If we consider using 'yarn audit fix', it could potentially be executed during the creation of a PR via a GitHub Action.

Reply:
We should fix Dependabot access. Taking this privately.

Reply:
I removed `yarn-audit-fix`. Less is more.